demos/storybook/stories/dropdown-toolbar/with-basic-usage.tsx

```diff
@@ -25,6 +27,10 @@ export const withBasicUsage = (): StoryFnReactReturnType => {
                 title={text('title', 'Title')}
                 subtitle={text('subtitle', 'Subtitle')}
                 menuGroups={menuGroups}
+                MenuProps={{
+                    anchorOrigin: { horizontal: direction === 'rtl' ? 'right' : 'left', vertical: 'bottom' },
+                    transformOrigin: { horizontal: direction === 'rtl' ? 'right' : 'left', vertical: 'top' },
+                }}
```

Comment:
This should live in the component so that and end-user's application will get this change automatically.

You can use
```
const theme = useTheme();
cont rtl = theme.direction === 'rtl';
```
and then conditionally set the origins based on that